reward/staking_info.go

```diff
@@ -198,26 +198,72 @@ func newEmptyStakingInfo(blockNum uint64) *StakingInfo {
 	return stakingInfo
 }
 
-func newStakingInfo(bc blockChain, helper governanceHelper, blockNum uint64, nodeAddrs []common.Address, stakingAddrs []common.Address, rewardAddrs []common.Address, KCFAddr common.Address, KFFAddr common.Address) (*StakingInfo, error) {
-	intervalBlock := bc.GetBlockByNumber(blockNum)
-	if intervalBlock == nil {
-		logger.Trace("Failed to get the block by the given number", "blockNum", blockNum)
-		return nil, errors.New(fmt.Sprintf("Failed to get the block by the given number. blockNum: %d", blockNum))
+func newStakingInfo(bc blockChain, helper governanceHelper, blockNum uint64, types []uint8, addrs []common.Address, effectiveStakings ...*big.Int) (*StakingInfo, error) {
```

Comment:
```suggestion
// newStakingInfo fills stakingAmount from effectiveStakings if set. Otherwise, stakingAmount is filled with the balances of staking contracts.
func newStakingInfo(bc blockChain, helper governanceHelper, blockNum uint64, types []uint8, addrs []common.Address, effectiveStakings ...*big.Int) (*StakingInfo, error) {
```

Reply:
Thanks. Applied at [52a73ab](https://github.com/klaytn/klaytn/pull/2158/commits/52a73ab9bb04af8d48ad503d7e1a07dad1af6d85)